.github/workflows/tests.yml

```diff
@@ -45,3 +45,67 @@ jobs:
       - name: Run unit tests
         run: ansible-test units --venv
         working-directory: ansible_collections/ansible/eda
+
+  integration-tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: ["3.10"]
+
+    env: 
+      EDA_CONTROLLER_HOST: "https://localhost:8443"
+      EDA_CONTROLLER_USERNAME: "admin"
+      EDA_CONTROLLER_PASSWORD: "testpass"
+      EDA_CONTROLLER_VERIFY_SSL: "False"
+
+    steps:
+      - name: Checkout ansible/eda
+        uses: actions/checkout@v3
+        with:
+          repository: sarathbrp/event-driven-ansible
+          path: ansible_collections/ansible/eda
+          ref: patch-1
+
+      - name: Checkout ansible/eda-server
+        uses: actions/checkout@v3
+        with:
+          repository: ansible/eda-server
+          path: eda-server # Note: Specify a separate path for this repository
+       
+      - name: Setup Docker Engine
```

Comment:
Are you sure these steps are necessary? The ubuntu-latest image already includes Docker.

Reply:
not required, removed.